### DISCUSSION TITLE
About the bug of tool sql-schema-to-go-struct

### DISCUSSION THREAD

**Discussion by Super-Tutu** [he3-app/he3-feedback#138] [Bug Report]
### Version
sql-schema-to-go-struct 1.0.0

### Environment
Windows v1.3.2

### Steps to reproduce

`CREATE TABLE `product` (
	`id` BIGINT(20) NOT NULL AUTO_INCREMENT COMMENT 'Id',
	`parent_cat_id` BIGINT(20) NULL DEFAULT NULL COMMENT '上级分类ID',
	`cat_id` BIGINT(20) NULL DEFAULT NULL COMMENT '分类ID',
	`brand_id` BIGINT(20) NULL DEFAULT NULL COMMENT '品牌id',
	`title` VARCHAR(200) NULL DEFAULT NULL COMMENT '商品名称' ,
	`short_title` VARCHAR(100) NULL DEFAULT NULL COMMENT '商品短名称' ,
	`degree` VARCHAR(100) NULL DEFAULT NULL COMMENT '酒精度数' ,
	`net_weight` VARCHAR(100) NULL DEFAULT NULL COMMENT '净含量' ,
	`price` DECIMAL(10,2) NULL DEFAULT '0.00' COMMENT '售价',
	`stock` INT(10) NULL DEFAULT '0' COMMENT '库存',
	`sale_num` INT(11) NULL DEFAULT '0' COMMENT '销量',
	`sort` INT(11) NULL DEFAULT '0' COMMENT '排序',
	`is_use` TINYINT(2) NULL DEFAULT '0' COMMENT '是否上架(1上架0下架)',
	`img` VARCHAR(200) NULL DEFAULT NULL COMMENT '商品主图' ,
	`cellar_img` VARCHAR(200) NULL DEFAULT NULL COMMENT '缩略图',
	`banner_img` JSON NULL DEFAULT NULL COMMENT '商品轮播图',
	`detail` TEXT NULL DEFAULT NULL COMMENT '商品详情',
	`created_at` DATETIME NULL DEFAULT NULL COMMENT '创建时间',
	`updated_at` DATETIME NULL DEFAULT NULL COMMENT '更新时间',
	PRIMARY KEY (`id`) USING BTREE
)
COMMENT='商品信息表'
ENGINE=InnoDB
AUTO_INCREMENT=10
;
`
![image](https://user-images.githubusercontent.com/24802126/232359646-4f388ff4-da57-4f95-b843-4be7343a5486.png)

### What is expected?
期望类似于下图中，结构体后跟随字段备注及结构体名称自动变更为相应的表名
![image](https://user-images.githubusercontent.com/24802126/232359685-592832a1-ce6c-4e64-93e8-0960696b4e54.png)

**Comment by Super-Tutu** [comment]
![image](https://user-images.githubusercontent.com/24802126/232359990-bc6e771b-cde1-4125-840a-548b4fd1742e.png)

**Comment by lyzhang1999** [comment]
感谢反馈，已经反馈给了产品部门：https://github.com/he3-app/he3-feedback/issues/139
